Zoom/Zoom7zUnarchiver.py

```diff
@@ -0,0 +1,175 @@
+#!/usr/bin/python
+#
+# Copyright 2010 Per Olofsson
```

Comment:
Indeed, quick fix as said in PR. And as always with crowd based source code, I don't know what to do with copyright. Most code in this file hasn't be written by me.